setup.py

```diff
@@ -97,7 +97,7 @@
     'pyyaml==3.11',
     'requests==2.9.1',
     'scikit-learn==0.17.1',
-    'ipykernel==4.4.1',
+    'ipykernel<=4.5.2',
```

Comment:
Well there's no reason why this package shouldn't work for you if you're installing it on an earlier version of `notebook`, which has an earlier version of `ipykernel`.
However, looking at the dependency of `notebook` on `ipykernel`, I see it's unbound to a certain version, so we should be fine here. Even if you have an older `notebook` version, it's fine that `pydatalab` updates your `ipykernel`. I'll pin it to `4.5.2`.